kornia/x/callbacks.py

```diff
@@ -72,35 +100,57 @@ class ModelCheckpoint:
     Args:
         filepath: the where to save the mode.
         monitor: the name of the value to track.
-
+        max_mode: if true metric will be multiply by -1
+                  turn this flag when increasing metric value is expected for example Accuracy
     **Usage example:**
 
     .. code:: python
+        def my_evaluate(self) -> dict[str, AverageMeter]:
```

Comment:
for some reason the doctests error is because of this: https://github.com/kornia/kornia/actions/runs/8110317553/job/22167347988?pr=2820

i guess because you changed the signature of the Callback ?

Reply:
It can be.
I should use Dict imported from typing over dict[] as type. As it won't work with older python versions will try it and ping if it will be w fix

Reply:
Ok I just move to older docstring version don't know why it didn't compile before but it should work fine right now